docs/content/integrations/bigquery/reference.mdx

```diff
@@ -117,15 +116,15 @@ def iris_data_partitioned(context) -> pd.DataFrame:
     full_df = pd.read_csv(
         "https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data",
         names=[
-            "Sepal length (cm)",
-            "Sepal width (cm)",
-            "Petal length (cm)",
-            "Petal width (cm)",
-            "Species",
+            "sepal_length_cm",
+            "sepal_width_cm",
+            "petal_length_cm",
+            "petal_width_cm",
+            "species",
```

Comment:
These renames are just clean up, correct? Not related to the core change here.

Reply:
the original code actually never worked (not sure how i missed it) since parentheses are invalid bigquery characters. But yet, it's not related to the pydantic config changes

Reply:
got it makes sense